week-1-ts-ecosystem/w01e01-typescript-basics/package.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "w01e01-typescript-basics",
   "version": "1.0.0",
-  "scripts": {}
+  "scripts": {
+    "build": "npx tsc src/main.ts",
```

Comment:
`npx` nie jest tutaj potrzebne, można usunąć, po tym jak zainstalujesz `typescript` to polecenie `tsc` jest dostępne lokalnie nie potrzeba `npx`